### PR TITLE
Test scope

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

Use as wide scope as possible. 

### Changed

- Fixture scope to session for as any as possible

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b [THIS-BRANCH-NAME] -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/apps/tower/conftest.py [hunk 1]
```diff
@@ -29,19 +29,19 @@ class CaseIDs:
     COMPLETED: str = "cuddlyhen_completed"
 
 
-@pytest.fixture(name="tower_id")
+@pytest.fixture(name="tower_id", scope="session")
 def fixture_tower_id() -> str:
     """Return a NF Tower id."""
     return TOWER_ID
```
Comment (seallard) [inline tests/apps/tower/conftest.py:35]: These fixtures are cheap to setup, but it does not hurt to default to a session wide scope :)

**Review by seallard — APPROVED** [review]
Nice 👍

File: tests/apps/tower/conftest.py (hunk 1 above)
Comment (henrikstranneheim) [reply tests/apps/tower/conftest.py:35]: Yes and I also think it is good to start being explicit about this

**Comment by henrikstranneheim** [comment]
Nothing to deploy

**Pull request merged**